### PR TITLE
Remington rulez

### PR DESCRIPTION
**Pull request by RDixon540**

### CODE REVIEW

**Review by ryan-schlientz — CHANGES_REQUESTED** [review]
I don't like that!

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+
+jjjj
```
Comment (ryan-schlientz) [inline .gitignore]: This line needs to be changed

**Review by ryan-schlientz — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ryan-schlientz) [inline .gitignore]: This is fine.

**Review by ryan-schlientz — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Kobs21) [inline .gitignore]: I agree

**Pull request closed without merging**